applications/plugins/SofaPython/SofaPython_test/PythonFactory_test.cpp

```diff
@@ -156,13 +156,13 @@ std::vector<std::vector<std::string>> dataconversionvalues =
      {"'aString'", "aString"},
      {"'aString'.join('[ ]')", "[aString aString]"},
      {"' '.join(['AA', 'BB', 'CC'])", "AA BB CC"},
-     {"[1, 2, 3, 4]", "1 2 3 4"},
-     {"[1.0, 2.0, 3.0, 4.0]", "1.0 2.0 3.0 4.0"},
-     {"['ab', 'cd', 'ef', 'gh']", "ab cd ef gh"},
-     {"[[1,2], [3,4], [5,6]]", "1 2 3 4 5 6"},
-     {"[['aa','bb'], ['cc','dd'], ['ee','ff']]", "aa bb cc dd ee ff"},
-     {"range(1,5)", "1 2 3 4"},
-     {"xrange(1,5)", "1 2 3 4"},
+     {"[1, 2, 3, 4]", "[1, 2, 3, 4]"},
+     {"[1.0, 2.0, 3.0, 4.0]", "[1.0, 2.0, 3.0, 4.0]"},
+     {"['ab', 'cd', 'ef', 'gh']", "['ab', 'cd', 'ef', 'gh']"},
+     {"[[1,2], [3,4], [5,6]]", "[[1, 2], [3, 4], [5, 6]]"},
+     {"[['aa','bb'], ['cc','dd'], ['ee','ff']]", "[['aa', 'bb'], ['cc', 'dd'], ['ee', 'ff']]"},
+     {"range(1,5)", "[1, 2, 3, 4]"},
+     {"xrange(1,5)", "xrange(1, 5)"},
```

Comment:
In the past, xrange was evaluated by the createObject function because it was implementing the Sequence operator. I see no reason why this should change.